### PR TITLE
Add samples for opening and saving Word document in Dropbox cloud storage

### PR DESCRIPTION
**Pull request by KonduruKeerthi**
Add samples for opening and saving Word document in Dropbox cloud storage

### CODE REVIEW

File: Read-and-Save-document/Open-and-save-Word-document/DropBox-Cloud-Storage/Open-Word-document/Open-Word-document/Controllers/HomeController.cs [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        /// Download file from DropBox cloud storage
+        /// </summary>
+        /// <returns></returns>
+        public async Task<MemoryStream> GetDocumentFromDropBox()
+        {
+            //Define the access token for authentication with the Dropbox API
+            var accessToken = "Access_Token";
+
+            //Define the file path in Dropbox where the file is located
+            var filePathInDropbox = "FilePath";
```
Comment (MohanaselvamJothi) [inline Read-and-Save-document/Open-and-save-Word-document/DropBox-Cloud-Storage/Open-Word-document/Open-Word-document/Controllers/HomeController.cs:74]: //Define the file path in Dropbox where the file is located
//For ex: "/WordTemplate.docx"

File: Read-and-Save-document/Open-and-save-Word-document/DropBox-Cloud-Storage/Save-Word-document/Save-Word-document/Controllers/HomeController.cs [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+        }
+        /// <summary>
+        /// Upload file to Dropbox
+        /// </summary>
+        /// <param name="stream"></param>
+        /// <returns></returns>
+        public async Task<MemoryStream> UploadDocumentToDropBox(MemoryStream stream)
+        {
+            //Define the access token for authentication with the Dropbox API
+            var accessToken = "Access_Token";
```
Comment (MohanaselvamJothi) [inline Read-and-Save-document/Open-and-save-Word-document/DropBox-Cloud-Storage/Save-Word-document/Save-Word-document/Controllers/HomeController.cs:270]: Add file path as separate variable as like download code

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: Read-and-Save-document/Open-and-save-Word-document/DropBox-Cloud-Storage/Open-Word-document/Open-Word-document/Controllers/HomeController.cs (hunk 1 above)
Comment (KonduruKeerthi) [reply Read-and-Save-document/Open-and-save-Word-document/DropBox-Cloud-Storage/Open-Word-document/Open-Word-document/Controllers/HomeController.cs:74]: Modified the comments as suggested.

File: Read-and-Save-document/Open-and-save-Word-document/DropBox-Cloud-Storage/Save-Word-document/Save-Word-document/Controllers/HomeController.cs (hunk 2 above)
Comment (KonduruKeerthi) [reply Read-and-Save-document/Open-and-save-Word-document/DropBox-Cloud-Storage/Save-Word-document/Save-Word-document/Controllers/HomeController.cs:270]: Modified the code as suggested.

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**